runtime/bcverify/bcverify.c

```diff
@@ -1204,6 +1206,8 @@ printMethod (J9BytecodeVerificationData * verifyData)
 				printf( "long");
 				break;
 		
+			case 'Q':
+				printf( "flattenable"); /* fallthrough*/
```

Comment:
is a space needed after "flattenable", i.e "flattenable "

Reply:
A space was needed. Added.

Reply:
where you referring to line 1152? this one still doesn't have a space

Reply:
Seems I did not check if line 1210 needs a space, just 1152. Will revisit.

Reply:
Space added